spec/spec_helper.rb

```diff
@@ -3,6 +3,7 @@
 require "webmock/rspec"
 require "timecop"
 require "govuk_sidekiq/testing"
+require 'rspec/its'
```

Comment:
can you maintain the double quote consistency here? (our coding standards rules regarding quotes are annoyingly loose but generally it's maintain convention of the file, on new files preference towards double quotes)